scripts/specs/format.ts

```diff
@@ -1,9 +1,11 @@
 import fsp from 'fs/promises';
 
+import oas2har from '@har-sdk/oas';
```

Comment:
```suggestion
import { oas2har } from '@har-sdk/oas';
```
since you only use 1 function